packages/block-editor/src/components/block-list/index.js

```diff
@@ -72,18 +71,17 @@ function BlockList( {
 		hasMultiSelection,
 		enableAnimation,
 	} = useSelect( selector, [ rootClientId ] );
-	const ref = useRef();
-	const onSelectionStart = useMultiSelection( { ref, rootClientId } );
```

Comment:
Notice that this is no longer rendered for every block list, but rather only for the root block list.